docs/_plugins/shortcodes.cjs

```diff
@@ -192,6 +192,20 @@ ${content.trim()}
 `;
   });
 
+  function dedent(str) {
```

Comment:
if we need this, raise it to module root.

alternately, the `dedent` npm package is available in package-lock, so just use that.

```
❯ npx npm-why dedent                                                                               (base) 

  Who required dedent:

  red-hat-design-system > @patternfly/pfe-tools > dedent@0.7.0

```